go-controller/pkg/util/net_unit_test.go

```diff
@@ -238,43 +238,102 @@ func TestIPFamilyName(t *testing.T) {
 }
 
 func TestMatchIPFamily(t *testing.T) {
+	tests := []struct {
+		desc        string
+		inpIPFamily bool // true matches with IPv6 and false with IPv4
```

Comment:
Could you change the field name `inpIPFamily` to something more meaningful? Perhaps: `isIPv6`/`lookupIPv6`?

Reply:
And below in `TestMatchIPNetFamily`